samples/contrib/aws-samples/simple_train_pipeline/training-pipeline.py

```diff
@@ -53,7 +53,7 @@ def training(
         instance_count=1,
         volume_size=50,
         max_run_time=3600,
-        model_artifact_path='s3://kubeflow-pipeline-data/mnist_kmeans_example/data',
+        model_artifact_path='s3://kubeflow-pipeline-data/mnist_kmeans_example/output',
```

Comment:
Kartik, just found today that line 37 need to change to s3://kubeflow-pipeline-data/mnist_kmeans_example/`train_data`